Client/Frontend/AuthenticationManager/AuthenticationSettingsViewController.swift

```diff
@@ -0,0 +1,166 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Eventually there is going to be other view controllers such as the passcode configuration and interval screens. I guess it depends on how we want to group up the classes. We can either make it revolve a theme (Authentication) or have it grouped by function (Settings, Views, ViewControllers).